packages/common/core-utils/.eslintrc.cjs

```diff
@@ -22,6 +22,9 @@ module.exports = {
 				// Test files are run in node only so additional node libraries can be used.
 				"import/no-nodejs-modules": ["error", { allow: ["node:assert", "node:process"] }],
 
+				// TODO: This rule is disabled in version 5.2+ of the eslint config, so can be removed when we update to 5.2+.
+				"import/order": "off",
+
```

Comment:
Common disagreement has been about index.js and sibling file imports. That is often a mistake.
Something different going on here? What files should be scrutinized?

Reply:
I removed the lint disable and worked around the complaints by grouping things together.